### PR TITLE
Fixed all 4 buttons for Wide-SCreens

### LINKED ISSUE #15711: UI on wide screens
Some UI controls are aligned to the right corner of the window, so they are too far from the main UI elements on wide screens.

Please align them to the left corner of the window, or make alignment configurable in the settings.

The main easily fixable buttons are:

### Browse
"Bookmark this SQL query" button.
SQL query links (Profile, Edit inline, Edit, Explain SQL, etc.)

### Structure
**Table structure** -> "Partition table" button
**Relation view** -> "Preview SQL" and "Save" buttons

Screenshot_2

### PR DESCRIPTION
**Pull request by rajat315315**
Signed-off-by: Rajat Jain <rajatjain.ix@gmail.com>

### Description

Fixes few buttons for Ultra-Wide screens.

Fixes #15711

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

File: themes/bootstrap/scss/_common.scss [hunk 1]
```diff
@@ -487,7 +487,6 @@ fieldset.tblFooters {
   margin-bottom: 0.5em;
   // avoid a thick line since this should be used under another fieldset
   border-top: 0;
-  text-align: $right;
```
Comment (williamdes) [inline themes/bootstrap/scss/_common.scss:490]: why was it necessary to remove the default behavior ?

Comment (rajat315315) [reply themes/bootstrap/scss/_common.scss]: Oops! by mistake!

Comment (rajat315315) [reply themes/bootstrap/scss/_common.scss]: Reversed!

**Review by williamdes — APPROVED** [review]
I will test it and merge it, but it looks good to me

File: themes/bootstrap/scss/_common.scss [hunk 2]
```diff
@@ -3801,3 +3801,10 @@ body .ui-dialog {
 .ui-timepicker-div dl .ui_tpicker_timezone select {
   margin-left: 50px;
 }
+
+// Extra large devices (large desktops, 1200px and up)
+@include media-breakpoint-up(xl){
+  .tblFooters {
```
Comment (williamdes) [inline themes/bootstrap/scss/_common.scss:3807]:
```suggestion
  fieldset.tblFooters,
  .tblFooters {
```

I added it in the merge

**Pull request merged**